### ISSUE TITLE
after gatk merge gvcf to vcf; then bcftools stats, I found a result of stats lose some count

### ISSUE THREAD

**Issue by duolin1986** [samtools/bcftools#1547]
as like title, if one line like this "**chr1    11140624        .       C       A,\*     366.64  .       AC=1,1;AF=0.250,0.250;AN=4;BaseQRankSum=-8...**" , in "A,*", symbol star indicate there is one sample lose REF C, So I think this line should increment indel counter, But actually Not, could you fix this problem? Thankyou

**Comment by pd3** [comment]
I don't think that would be correct. It is the same indel reported upstream, not another indel. Or maybe I am misunderstanding?
